content/klc/S4.6.adoc

```diff
@@ -1,14 +1,15 @@
 +++
-brief = "Hidden pins"
+brief = "Symbol pins must not be hidden unless they are not connected, part of a stack or power symbols"
 +++
 
-Hidden connection pins are generally not allowed for schematic symbols. Any connection point must be visible, otherwise unexpected net connections can occur.
+Hidden symbol pins are generally not allowed for schematic symbols. Any connection point must be visible, otherwise unexpected net connections can occur.
 
-Hidden power pins are also not allowed.
 
 **Exceptions**
 
-. Unused NC pins can be set to invisible. In this case the electrical type must be `Not Connected`. The end of the pin should lie on the symbols outline box to prevent unwanted connections to the invisible pin (see screenshot below).
+. Power input pins must not be invisible unless used in power symbols. (Hidden power input pins are global labels.) Refer to the link:/libraries/klc/S4.3[requirements for pin stacking]
```

Comment:
Punctuation is a bit odd on this sentence.
Otherwise, I think this PR looks OK.

Reply:
The reason for the strange punctuation might be because i tried to use this strange RFC defined standardized language. https://github.com/KiCad/kicad-website/issues/289

Do you have a better suggestion?

Reply:
Something like this maybe?

Power input pins must not be invisible unless used in power symbols (hidden power input pins are global labels). Refer to the link:/libraries/klc/S4.3[requirements for pin stacking].

Reply:
Am i blind or is yours word for word the same?

Reply:
The wording is the same. It's just the punctuation and capitalisation.

Reply:
fixed (forgot to mark it as such)